en:
  not_found: "%{model_name} with %{primary_key}=%{id} does not exist"
  conflict: "%{model_name} with %{primary_key}=%{id} already exists"
  not_found.no_chores: "You are not assigned to any chores of type %{chore_type_id} for week %{week_id}"
  bad_request.already_completed: "%{model_name} with week_id=%{week_id} and chore_type_id=%{chore_type_id} is already completed"
  bad_request.transfer_multiple_users: Cannot transfer chore to multiple users
  forbidden.transfer_other_user: You cannot %{action} a transfer for another user
  bad_request.transfer_to_self: Cannot transfer chore to the same user
  bad_request.transfer_no_chores: "No chores of type %{chore_type_id} for week %{week_id} assigned to user with id=%{user_id}"
  bad_request.transfer_completed: Transfer is already completed
